hr_department_sequence/models/hr_department.py

```diff
@@ -32,13 +32,8 @@ class HrDepartment(models.Model):
     sequence = fields.Integer(string='Sequence', index=True,
                               help="Gives the sequence order when displaying "
                               "a list of departments.")
-    parent_id = fields.Many2one(ondelete='restrict')
-    parent_left = fields.Integer(string='Left Parent', index=True)
-    parent_right = fields.Integer(string='Right Parent', index=True)
-    child_ids = fields.One2many(comodel_name='hr.department',
-                                inverse_name='parent_id',
-                                string='Children Departments')
-    active = fields.Boolean(string='Active', default=True)
```

Comment:
Why remove these fields?

Reply:
Because this fields are already in Odoo core  https://github.com/OCA/OCB/blob/5654fe4dd3c4ae55fe13bac89cd1790b6d1593af/addons/hr/models/hr.py#L280

Reply:
ok